### PR TITLE
[Milestone 1] Creating Model

### PR DESCRIPTION
**Pull request by 41hulk**
### What does this PR do?
- Build your project schema.
    - Create and run the necessary migration files.
    - Table and column names should match the ERD diagram.
        - note: photo for users table should be a link to an image
    - Foreign keys should be included.
    - All columns that are foreign keys should have a corresponding index.

### CODE REVIEW

**Comment by ahmetbozaci** [comment]
# ❌ ❌ ❌ STATUS: INVALID  ❌ ❌ ❌ 


### Hello @41hulk  ✋🏼 
#### You have submitted a project, that is using GitHub flow. You should use git-flow. Kindly change it.

#### Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏


### Cheers and Happy coding! 💻 👏
coding


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._

_Invalid Code Review Request **does not count** into the code reviews limit._

___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

File: db/schema.rb [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema[7.0].define(version: 2022_09_17_125519) do
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
+  create_table "comments", force: :cascade do |t|
+    t.string "text"
```
Comment (ahmetbozaci) [inline db/schema.rb]: - [ ] The text type should be text. Kindly change it.

File: db/schema.rb [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    t.bigint "posts_id", null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["author_id"], name: "index_likes_on_author_id"
+    t.index ["posts_id"], name: "index_likes_on_posts_id"
+  end
+
+  create_table "posts", force: :cascade do |t|
+    t.string "title"
+    t.string "text"
```
Comment (ahmetbozaci) [inline db/schema.rb]: - [ ] The text type should be text. Kindly change it.

File: db/schema.rb [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.bigint "author_id", null: false
+    t.index ["author_id"], name: "index_posts_on_author_id"
+  end
+
+  create_table "users", force: :cascade do |t|
+    t.string "name"
+    t.string "photo"
+    t.string "bio"
```
Comment (ahmetbozaci) [inline db/schema.rb]: - [ ] The bio type should be text. Kindly change it.

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️


 @41hulk  ✋🏼 

#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Highlights 💯
✔️ Your git-flow is correct. 
✔️ You set up linter files and no errors. 
✔️ Your pull request is very professional. It has a meaningful title and summary. Keep it up.
___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED ✔️ 🟢 

## Hi @41hulk 👋, 

Well done for implementing the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**